src/components/tooltip/bl-tooltip.css

```diff
@@ -3,34 +3,18 @@
   cursor: pointer;
 }
 
-.tooltip {
-  position: fixed;
-  box-sizing: border-box;
-  border: none;
-  background-color: var(--bl-color-neutral-darker);
-  color: var(--bl-color-neutral-full);
-  border-radius: var(--bl-size-3xs);
-  pointer-events: none;
-  font: var(--bl-font-title-3-regular);
-  padding: var(--bl-size-m);
-  z-index: var(--bl-index-tooltip);
-  width: max-content;
-  hyphens: auto;
-
-  --max-viewport: calc(100vw - var(--bl-size-s));
-  --tooltip-max-width: 424px;
-
-  max-width: min(var(--max-viewport), var(--tooltip-max-width));
+.wrapper{
+  display: inline-block;
+  color: var(--bl-color-neutral-full);;
 }
 
-.tooltip:not(.visible) {
-  visibility: hidden;
+bl-popover {
+  --bl-popover-background-color: var(--bl-color-neutral-darker);
+  --bl-popover-arrow-display: block;
+  --bl-popover-border-size: calc(0 * 1px);
```

Comment:
The current implementation of popover doesn't accept unitless border-size (like `0`). And our linter doesn't accept `0px` since it's a best practice to write `0` without unit. I asked for a solution to this on stackoverflow: https://stackoverflow.com/questions/76179826/css-calc-with-a-variable-that-can-be-unitless

Reply:
With the answers to SO question, I come up with the idea to change our linter to accept (even suggest) using units with 0 value as this article suggests: https://www.oddbird.net/2022/08/04/zero-units/